src/style.css

```
@@ -0,0 +1,98 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: normal;
+  font-size: 600;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+}
+
+.search-box {
+  display: block;
+  align-items: center;
+  position: relative;
+}
+
+.icon {
+  position: absolute;
+  padding-left: 26px;
+  padding-top: 25px;
+  width: 45px;
+  height: 44px;
+}
+
+.input {
+  width: 100%;
+  height: 70px;
+  padding: 0 0 0 62px ;
+  text-align: left;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: inherit;
+  font-weight: 300;
+  font-size: 16px;
```

```suggestion
  font-size: 16px;
  outline: none;
```
you should add `outline` to remove the default border from the input